acceptance-tests/Makefile.am

```diff
@@ -2425,4 +2445,13 @@ $(CORECLR_PATH)%.exe: $(CORECLR_PATH)%.cs
 test-runner.exe: $(top_srcdir)/mono/tests/test-runner.cs
 	$(MCS) -debug -out:$@ $<
 
-CLEANFILES = $(CORECLR_TESTSI_CS) $(CORECLR_TESTSI_IL) *.dll *.exe *.mdb
+Tests:
+	mkdir Tests
```

Comment:
I actually found a workaround: there is a BVT_ROOT env variable which is used in the code to change the folder where to find the tests.